tools/scripts/post_webhook.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+
+# You can lookup sample webhook events online.
+#    GitLab - https://docs.gitlab.com/ee/user/project/integrations/webhook_events.html
+#    GitHub - https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads
+#
+
+# User modified variables (maybe params?)
```

Comment:
> (maybe params?)

old comment? Looks like they are implemented as CLI params now.

Also might be nice to reformat these example values into the CLI use.

Reply:
Good catch, that comment should have been completely removed; they were my actual variables before I added click. But, I do like the idea of adding example calls so I added that to the --help. I also moved the comment about where to get more files to the help.